copilot-chat-frontend.el

```diff
@@ -54,6 +56,9 @@
 Each element must be a `copilot-chat-frontend' struct instance.
 Elements are added in the module that defines each front end.")
 
+(defvar copilot-chat--frontend-init-p nil
+  "Flag to indicate if the frontend has been initialized.")
+
```

Comment:
Should be frontend instance field.
Instead global var.

Reply:
I added `instance-init-fn` for instance setup.